lbrynet/dht/node.py

```diff
@@ -190,6 +201,12 @@ def printContacts(self, *args):
                 print "    %s:%i" % (contact.address, contact.port)
         print '=================================='
 
+    def hasContacts(self):
+        for i in range(len(self._routingTable._buckets)):
+            if len(self._routingTable._buckets[i]._contacts) > 0:
+                return True
+        return False
```

Comment:
Less complicated and maybe easier to read:
```python
    def hasContacts(self):
         for _ in self._routingTable._buckets:
             if self._routingTable._buckets[i]._contacts:
                 return True
         return False
```

Reply:
Thanks! I overlooked that `list()` evaluates to False when empty.

About the external for: I copy/pasted from printContacts() :) maybe we should clean that too (and not only that)

Reply:
change applied

Reply:
I ended up using this simpler version:
```python
204     def hasContacts(self):
205         for bucket in self._routingTable._buckets:
206             if bucket._contacts:
207                 return True
208         return False
```